web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -26,6 +26,23 @@
         <label for="integration-url-topic-input">{{t "Topic"}}</label>
         <input type="text" id="integration-url-topic-input" class="modal_text_input integration-url-parameter" maxlength="{{ max_topic_length }}" />
     </div>
+    <div id="integration-events-toggle-wrapper" class="input-group hide">
+        <label class="checkbox">
+            <input type="checkbox" id="show-integration-events"/>
+            <span></span>
+        </label>
+        <label class="inline" for="show-integration-events">
+            {{t "Filter events that will trigger notifications?"}}
+        </label>
+    </div>
+    <div class=" input-group hide" id="integration-events-ui">
+        <label for="integration-events-container">{{t "Events to include:"}}</label>
+        <div class="integration-checkbox-button-container">
```

Comment:
I'd rename this class to "integration-all-events-buttons".

And I'd rename the two button ids to be:
- "add-all-integration-events"
- "remove-all-integration-events".